src/oc/web/lib/utils.cljs

```diff
@@ -636,6 +636,10 @@
 
 (def default-headline "Untitled post")
 
+(def default-abstract "Post preview: summarize why this post matters…")
+
+(def max-abstrct-length 280)
```

Comment:
@bago2k4 this isn't used anywhere

Reply:
it used in `oc.web.components.cmail;569`